### PR TITLE
feat(locales): add locales implementation

### PR DESCRIPTION
**Pull request by Satont**
### TODO:
systems: 
- [x] commands locales
- [x] events locales
- [x] keywords locales 
- [x] variables locales
- [x] timers locales
- [x] overlays locales
---
settings:
- [ ] moderation locales 
- [ ] users locales
- [ ] notables players locales 
---
integrations:
- [ ] DonationAlerts locales
- [ ] Spotify locales
- [ ] Qiwi Donate locales
- [ ] StreamLabs locales

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "plugins": ["@babel/plugin-proposal-class-properties"]
+}
```
Comment (AlexWayfer) [inline .babelrc]: Oh, no newline at the and of file…

Maybe you should use editorconfig.org and plugin for editor with support.

File: app.js [hunk 2]
```diff
@@ -14,7 +14,6 @@ async function load () {
   require('./libs/panel')
 
   global.systems = await autoLoad('./systems/')
-
```
Comment (AlexWayfer) [inline app.js]: Change looks unnecessary.

File: libs/locales.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+const { readdirSync, readFileSync } = require('fs')
+const { join } = require('path')
+const { get } = require('lodash')
+const { io } = require('./panel')
+
+
```
Comment (AlexWayfer) [inline libs/locales.js]: Double empty lines? Maybe you want to use ESLint.

File: libs/locales.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    let query = get(this.lang, path, 'Not found')
+    if (variables) {
+      for (let variable of Object.entries(variables)) {
+        const regexp = new RegExp(`{${variable[0]}}`, 'ig')
+        query = query.replace(regexp, variables[variable[0]])
+      }
+    }
+    return query
+  }
+  async sockets () {
```
Comment (AlexWayfer) [inline libs/locales.js]: And here is no empty line at all…

File: locales/ru.json [hunk 5]
```diff
@@ -0,0 +1,8 @@
+{
+	"commands": {
+    "notFound": "Команда {name} не найдена.",
+    "ui": {
+      "create": "Создать команду"
+    }
+  }
+}
```
Comment (AlexWayfer) [inline locales/ru.json]: The same with newlines.

File: libs/locales.js (hunk 3 above)
Comment (Satont) [reply libs/locales.js]: Yes, i will setup eslint later. Thank you.

File: .babelrc (hunk 1 above)
Comment (Satont) [reply .babelrc]: Sure, i'll create new PR with code styling, eslint after that PR will be done.

File: app.js (hunk 2 above)
Comment (Satont) [reply app.js]: You are right, should be fixed now.

File: libs/locales.js (hunk 4 above)
Comment (Satont) [reply libs/locales.js]: Fixed. Nice point!

File: locales/ru.json (hunk 5 above)
Comment (Satont) [reply locales/ru.json]: I think json shouldn't contain newlines.

Comment (AlexWayfer) [reply locales/ru.json]: It's the rule for any text file from POSIX. You can read:

1. https://thoughtbot.com/blog/no-newline-at-end-of-file
2. https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file
3. https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Comment (Satont) [reply locales/ru.json]: Sure, i'll create new PR with eslint and EditorConfig after merge that PR.

**Comment by AlexWayfer** [comment]
> Satont closed this 12 hours ago

Why? :disappointed_relieved:

**Comment by Satont** [comment]
> > Satont closed this 12 hours ago
> 
> Why?

I will redo this in new PR will have motivation for that.

**Comment by AlexWayfer** [comment]
> I will redo this in new PR will have motivation for that.

"when"*?

**Comment by Satont** [comment]
@AlexWayfer yea, i mean when*.

**Pull request closed without merging**